conviva/package.json

```diff
@@ -21,7 +21,7 @@
     "bundle": "rollup -c rollup.config.mjs",
     "watch": "npm run bundle -- --watch",
     "build": "npm run clean && npm run bundle",
-    "serve": "http-server",
+    "serve": "http-server ./.. -o /conviva/test/pages/main_umd.html",
```

Comment:
Do we want to open the pages folder instead? Then they can choose our other pages more easily if they wanna test yospace for example.

Reply:
✅